### PR TITLE
Project happy thought api

### PR DESCRIPTION
**Pull request by destinyleia**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -3,14 +3,32 @@ import bodyParser from 'body-parser'
...
-//
-//   PORT=9000 npm start
+//The Mogoose model 
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: [true, "Hey, you need to write a message!"],
+    minLength: 5,
+    maxLength: 140,
+  },
```
Comment (KarolCislowski) [inline server.js:17]: Nicely structured data 👍🏻

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Your code looks very nice, but you need to provide links to both frontend and backend for testing purposes. 😸

**Comment by destinyleia** [comment]
Hi Karol,

Thanks for the review! 

Here's the link to the deployed site https://happy-thoughts-dess.herokuapp.com/thoughts

and heres to the frontend: https://happy-thoughts-happy-life.netlify.app/

Thank you! :D

File: server.js [hunk 2]
```diff
@@ -23,6 +41,37 @@ app.get('/', (req, res) => {
...
+      message: 'Could not post thought',
+      errors: error.errors
+    })
+  }
+})
+
+app.patch('thoughts/:id', async (req, res) => {
+  await Thought.updateOne({ _id: req.params.id }, req.body)
+  res.status(200)
+})
```
Comment (HIPPIEKICK) [inline server.js:73]: This endpoint doesn't correspond with your frontend. The logic should increment the hearts in your message/thought 👇
```
hearts: {
    type: Number,
    default: 0,
  },
```

Comment (destinyleia) [reply server.js:73]: How do I solve it? :) I have no idea... need help :)

Comment (HIPPIEKICK) [reply server.js:73]: Have you looked at Stack Overflow or the recorded material? 😇 Here's a link to all questions with the tag project-happy-thoughts-api -> https://stackoverflow.com/c/technigo/questions/tagged/189

Comment (destinyleia) [reply server.js]: It's solved (I think) :) Please review! :D

Comment (HIPPIEKICK) [reply server.js]: You've forgotten to update the frontend with the URL to your personal API, so no way to test it! The code looks good though

Comment (HIPPIEKICK) [reply server.js]: Although I can't like even in Postman. Maybe change the default to 0 instead of `() => 0`

Comment (HIPPIEKICK) [reply server.js]: Your frontend still looks like it has the technigo-url 🤔

**Pull request closed without merging**